cups/raster.c

```diff
@@ -1076,7 +1076,7 @@ cupsRasterWriteHeader(
 
     void *dst = fh.cupsReal; /* Bypass bogus compiler warning */
     void *src = r->header.cupsReal;
-    memcpy(dst, src, sizeof(fh.cupsReal) + sizeof(fh.cupsString));
+    memcpy(dst, src, sizeof(fh.cupsReal));
```

Comment:
This is wrong. We need to copy everything in cupsReal and cupsString - cupsString follows cupsReal in the structure so this is valid.